components/Belt/index.js

```diff
@@ -0,0 +1,23 @@
+import { Col, Tag } from "antd";
+import * as BELT from "~/lib/belt";
+
+import styles from "./style.module.css";
+
+function Belt({ belt }) {
+  return (
+    <>
+      {belt ? (
```

Comment:
I think this should exist, we do `!belt ? styles.nobelt : styles.capitalize` because when we do not have a badge it should display "Sem Cinturão" :reminder_ribbon:

Reply:
I've updated it. It looks like this now:
![image](https://user-images.githubusercontent.com/55213469/139847116-80e66f02-5069-44d8-bc77-f38fc7a9d061.png)

Is it good or should I change something?